recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ source:
       - patches/0009-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch
       - patches/0010-Win32-Do-not-download-externals.patch
       - patches/0011-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch
-      - patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
+      #- patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
```

Comment:
Why is this patch commented?

Reply:
this patch needs some fix, i first removed it to try it out, and CI passed. Now I'm trying to see how to fix the 0012 patch. I think some lines needs to be updated.

Reply:
looking at the 0012 patch file, it seems to be generated file. @jakirkham is there anything i should read about to fix this?

Reply:
It's probably something simple like more code was added to the file so the line numbers are off. Basically this if just trying to prioritize to search for libraries to load in the Conda environment. Though I could be wrong about this

This was Ray's patch. Have already cc'd him. Would give him a chance to respond